HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -6283,11 +6287,18 @@ def self.interpolate_seer2(seer2, eer2, seer2_array, seer2_eer2_ratio_array, cop
     clg_ap.cool_ff_chg_values = [26.67, 35.0]
 
     if cooling_system.is_a?(HPXML::HeatPump) && cooling_system.heat_pump_type == HPXML::HVACTypeHeatPumpGroundToAir
+      # Need a separate set of coefficients for installation quality
+      # Based on RESNET MINHERS Addendum 82
+      clg_ap.cool_cap_fflow_spec_iq = [0.718664047, 0.41797409, -0.136638137]
+      clg_ap.cool_eir_fflow_spec_iq = [1.143487507, -0.13943972, -0.004047787]
       clg_ap.cool_rated_shr_gross = 0.708
       clg_ap.cool_rated_cfm_per_ton = HVAC::RatedCFMPerTon
       return
     end
 
+    # Based on RESNET MINHERS Addendum 82
+    clg_ap.cool_cap_fflow_spec = [0.718664047, 0.41797409, -0.136638137]
+    clg_ap.cool_eir_fflow_spec = [1.143487507, -0.13943972, -0.004047787]
```

Comment:
We shouldn't duplicate values in code, that's not good practice. If they ever change in the future, it'd be easy to change one set and not realize there's a second set. Also, it's pretty confusing that the IQ code in hvac.rb and hvac_sizing.rb only sometimes uses the `foo_iq` variables.

I suggest:
1. Set the `foo_iq` variable up front for all system types. Only use these variables in the IQ code in hvac.rb and hvac_sizing.rb.
2. Then set `foo = foo_iq` for all non-GHP system types. These variables will be used for the model.

Reply:
This is good suggestion Scott! I was trying to resolve this and realized that it's not the best solution, wrote it to wish some quick fix to let it run. I'll change it so that it's not duplicated.